tests/integration/tier0/check-user-response/test_user_response.py

```diff
@@ -0,0 +1,50 @@
+from envparse import env
+
+
+def test_check_user_response(convert2rhel):
+
+    # Run c2r registration with no username and password provided
+    # check for user prompt enforcing input, then continue with registration
+    with convert2rhel(
+        "-y --no-rpm-va --serverurl {}".format(
+            env.str("RHSM_SERVER_URL"),
+        )
+    ) as c2r:
+        c2r.expect_exact(" ... activation key not found, username and password required")
+        c2r.expect_exact("Username: ")
+        c2r.sendline()
+        if c2r.expect_exact(["Username", "Password"]) == 0:
+            # Provide username, expect password prompt
+            c2r.sendline(env.str("RHSM_USERNAME"))
+        else:
+            c2r.sendcontrol("d")
```

Comment:
I don't quite understand why do you use `ctrl + d` here. Is there some reason for it?

Reply:
Good point, I will change that.